bazel/cargo/Cargo.toml

```diff
@@ -21,46 +15,46 @@ once_cell = "1.3"
 wasmtime = {version = "0.21.0", default-features = false}
 wasmtime-c-api-macros = {git = "https://github.com/bytecodealliance/wasmtime", tag = "v0.21.0", path = "crates/c-api/macros"}
 
-[raze]
+[package.metadata.raze]
 gen_workspace_prefix = "proxy_wasm_cpp_host_raze_"
```

Comment:
Actually, while you're here, would you mind changing this to `"proxy_wasm_cpp_host_raze"` or even `"proxy_wasm_cpp_host"`? I don't think the extra `_` at the end is necessary, and it results in the weird `proxy_wasm_cpp_host_raze__fetch_remote_crates()` instead of `proxy_wasm_cpp_host_raze_fetch_remote_crates()` or `proxy_wasm_cpp_host_fetch_remote_crates()`.

Reply:
that's true. Thanks for pointing out